apps/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,52 @@
-import { Component, OnInit } from '@angular/core';
+import { AsyncPipe } from '@angular/common';
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { map, Observable } from 'rxjs';
+import { CityStore } from '../../data-access/city.store';
+import {
+  FakeHttpService,
+  randomCity,
+} from '../../data-access/fake-http.service';
+import { Item } from '../../model/item.model';
+import { CardComponent } from '../../ui/card/card.component';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: `<app-card
+    [items]="cityItems$ | async"
+    customClass="bg-light-blue"
+    [listItemTemplate]="listItemTemplate"
+    (addNewItem)="addNewCity()"
+    (deleteItem)="deleteCity($event)">
+    <ng-container image>
+      <img src="assets/img/city.png" width="200px" />
+    </ng-container>
+    <ng-template #listItemTemplate let-item>{{ item?.name }} (C)</ng-template>
+  </app-card>`,
   standalone: true,
-  imports: [],
+  imports: [CardComponent, AsyncPipe],
+  changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class CityCardComponent implements OnInit {
-  constructor() {}
+  cityItems$!: Observable<Item[]>;
```

Comment:
you can set your observable here instead of declaring it and setting it in ngOnInit. :)